### PR TITLE
chore: Run integration tests on `CI`

### PR DESCRIPTION
**Pull request by antusus**

### CODE REVIEW

File: src/intTest/java/com/box/sdk/BoxRecentsIT.java [hunk 1]
```diff
@@ -9,11 +11,13 @@
 import java.util.Iterator;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
 
 /**
  * {@link BoxRecents} related integration tests.
  */
+@Ignore
```
Comment (mwwoda) [inline src/intTest/java/com/box/sdk/BoxRecentsIT.java:20]: I suppose that there is a problem with this test, right?

File: src/intTest/java/com/box/sdk/EventStreamIT.java [hunk 2]
```diff
@@ -6,8 +6,10 @@
 
 import java.util.concurrent.LinkedBlockingQueue;
 import java.util.concurrent.TimeUnit;
+import org.junit.Ignore;
 import org.junit.Test;
 
+@Ignore
```
Comment (mwwoda) [inline src/intTest/java/com/box/sdk/EventStreamIT.java:12]: I suppose that there is a problem with this test, right?

File: src/main/java/com/box/sdk/BoxFileVersionRetention.java [hunk 3]
```diff
@@ -67,6 +67,7 @@ public BoxFileVersionRetention(BoxAPIConnection api, String id) {
      * and {@link BoxRetentionPolicyAssignment#getFileVersionsUnderRetention(String...)} instead.
      * Retrieves all file version retentions matching given filters as an Iterable.
      */
+    @Deprecated
```
Comment (mwwoda) [inline src/main/java/com/box/sdk/BoxFileVersionRetention.java:70]: Is there any alternative to this method? Is it possible to point a client to it?

**Review by mwwoda — DISMISSED** [review]
Approving with small comments/questions.

File: src/main/java/com/box/sdk/BoxFileVersionRetention.java (hunk 3 above)
Comment (antusus) [reply src/main/java/com/box/sdk/BoxFileVersionRetention.java:70]: Yes, but it is documented in JavaDoc above the method:

```
* @deprecated This method will be deprecated in the future. Please use
* {@link BoxRetentionPolicyAssignment#getFilesUnderRetention(int, String...)}
* and {@link BoxRetentionPolicyAssignment#getFileVersionsUnderRetention(String...)} instead.
```

File: src/intTest/java/com/box/sdk/BoxRecentsIT.java (hunk 1 above)
Comment (antusus) [reply src/intTest/java/com/box/sdk/BoxRecentsIT.java]: I forgot to enable it. But it required user connection.

File: src/intTest/java/com/box/sdk/EventStreamIT.java (hunk 2 above)
Comment (antusus) [reply src/intTest/java/com/box/sdk/EventStreamIT.java]: I forgot to enable that.

**Review by mwwoda — DISMISSED** [review]

**Review by mwwoda — DISMISSED** [review]

**Pull request merged**